classes/ActionScheduler_LogEntry.php

```diff
@@ -4,12 +4,37 @@
  * Class ActionScheduler_LogEntry
  */
 class ActionScheduler_LogEntry {
-	protected $action_id = '';
-	protected $message = '';
+	protected $action_id =  '';
+	protected $message =  '';
+	protected $date;
 
-	public function __construct( $action_id, $message ) {
+	/**
+	 * Constructor
+	 *
+	 * @param mixed  $action_id	Action ID
+	 * @param string $message   Message
+	 * @param string $date
+	 */
+	public function __construct( $action_id, $message, $date  = '' ) {
 		$this->action_id = $action_id;
-		$this->message = $message;
+		$this->message   = $message;
+		$this->date      = $date;
```

Comment:
I think this should be stored as a `DateTime` object to be consistent regardless of the date format passed in. At the moment, its value could be any type of date/time string, e.g. `2017-09-20 20:00:00` or `January 1st, 2018` or something else entirely.